### PR TITLE
List GC members

### PR DESCRIPTION
**Pull request by Elchi3**

### CODE REVIEW

File: content/about.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ We use our funding to employ technical writers who work on web documentation pro
 
 Our Governing Committee administers the financial sponsorships made via [Open Collective](https://opencollective.com/open-web-docs), and explores strategic partnerships and needs for web documentation projects. Governing Committee members are representatives of organizations that have made substantial financial contributions to OWD, and include one representative from staff.
 
+- Coil
+- Google
+- Meta
+- Microsoft
+- Samsung (associate member)
```
Comment (robnyman) [inline content/about.md:33]: Daniel Appelquist (associate member)

File: content/about.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ We use our funding to employ technical writers who work on web documentation pro
...
+- W3C (associate member)
```
Comment (robnyman) [inline content/about.md:34]: Dominique Hazael-Massieux  (associate member)

**Review by wbamberg — APPROVED** [review]
I'm happy with this if @robnyman is.

**Comment by robnyman** [comment]
@wbamberg I am. :-)
And if there are other thoughts later, then we can just change it then.

**Pull request merged**